mmv1/products/compute/RegionResizeRequest.yaml

```diff
@@ -0,0 +1,413 @@
+      Name of the compute region scoping this request. Name should conform to RFC1035.
+    url_param_only: true
+    required: true
+    resource: 'Region'
+    imports: 'name'
+  - name: 'instanceGroupManager'
+    type: ResourceRef
+    description: |
+      The name of the regional managed instance group. The name should conform to RFC1035 or be a resource ID.
+      Authorization requires the following IAM permission on the specified resource instanceGroupManager:
+      *compute.instanceGroupManagers.update
+    url_param_only: true
+    required: true
+    resource: 'InstanceGroupManager'
+    imports: 'name'
+# Non-URL parameters including input and output parameters
+properties:
+  - name: 'creationTimestamp'
+    type: Time
+    description: |
+      The creation timestamp for this resize request in RFC3339 text format.
+    output: true
+  - name: 'state'
+    type: Enum
+    description: |
+      [Output only] Current state of the request.
+    output: true
+    enum_values:
+      - 'CREATING'
+      - 'ACCEPTED'
+      - 'FAILED'
+      - 'SUCCEEDED'
+      - 'CANCELLED'
+  - name: 'name'
+    type: String
+    description: |
+      The name of this resize request. The name must be 1-63 characters long, and comply with RFC1035.
+    required: true
+  - name: 'description'
+    type: String
+    description: |
+      An optional description of this resize-request.
+  - name: 'resizeBy'
+    type: Integer
+    description: |
+      The number of instances to be created by this resize request. The group's target size will be increased by this number.
+    required: true
+  - name: 'requestedRunDuration'
+    type: NestedObject
+    description: |
+      Requested run duration for instances that will be created by this request. At the end of the run duration instances will be deleted.
+    properties:
+      - name: 'seconds'
+        type: String
+        description: |
+          Span of time at a resolution of a second. Must be from 600 to 604800 inclusive. Note: minimum and maximum allowed range for requestedRunDuration is 10 minutes (600 seconds) and 7 days(604800 seconds) correspondingly.
+        required: true
+      - name: 'nanos'
+        type: Integer
+        description: |
+          Span of time that's a fraction of a second at nanosecond resolution. Durations less than one second are represented with a 0 seconds field and a positive nanos field. Must be from 0 to 999,999,999 inclusive.
+  - name: 'status'
+    type: NestedObject
+    description: |
+      [Output only] Status of the request.
+    output: true
+    properties:
+      # Status.error
+      - name: 'error'
+        type: NestedObject
+        description: |
+          [Output only] Fatal errors encountered during the queueing or provisioning phases of the ResizeRequest that caused the transition to the FAILED state. Contrary to the lastAttempt errors, this field is final and errors are never removed from here, as the ResizeRequest is not going to retry.
```

Comment:
```suggestion
          Fatal errors encountered during the queueing or provisioning phases of the ResizeRequest that caused the transition to the FAILED state. Contrary to the lastAttempt errors, this field is final and errors are never removed from here, as the ResizeRequest is not going to retry.
```

Reply:
Removed from all the places where it was mentioned. Check please.